test/sai_test/config/route_configer.py

```diff
@@ -739,6 +749,31 @@ def create_nexthop_by_rif(self, rif, nexthop_device: Device):
 
         return nhopv4, nhopv6
 
+    def create_nexthop_group_by_nexthops(self, dest_device: Device):
```

Comment:
for a method, the parameters can always to illustrate what kinds of data that will be needed in this method, is it only need the device?